free-programming-books.md

```diff
@@ -2166,7 +2166,7 @@ For resources on Angular, Backbone, D3, Dojo, Ember, Express, jQuery, Knockout,
 
 ### Self
 
-* [The Self Handbook](http://handbook.selflanguage.org/current/)
+* [The Self Handbook](http://handbook.selflanguage.org/)
```

Comment:
I thought I could fix it, but no. Just need to remove trailing slash from top-level url - http://handbook.selflanguage.org not http://handbook.selflanguage.org/

Reply:
I deleted my fork of the project, so I was unable to update this PR. New PR: #2591